### PR TITLE
Authentication (already merged, this is for review)

### PR DESCRIPTION
**Pull request by jakemoves**

### CODE REVIEW

File: admin-site/src/Button.svelte [hunk 1]
```diff
@@ -40,7 +41,7 @@ export let buttonHtml = "";
     <div class={gridStyle}>
     
             <button on:click
-                type="button" 
+                type={buttonType}
```
Comment (helloCaller) [inline admin-site/src/Button.svelte:44]: Ok great, nice use of default value.

File: admin-site/src/Login.svelte [hunk 2]
```diff
@@ -37,22 +40,49 @@
...
     }
   }
 
+  function onLoginButton(event){
+    event.preventDefault()
+
+    login()
+    .then( () => {
+      // happy path!
+      // a cookie named 'jwt' should now be present in this browser
```
Comment (helloCaller) [inline admin-site/src/Login.svelte:70]: I'm curious how a cookie is built and handled, I'm assuming that's happening on the server side?

File: src/cohort-app.js [hunk 3]
```diff
@@ -5,6 +5,8 @@
 const express = require('express');
 const bodyParser = require('body-parser')
 const path = require('path')
+const passport = require('passport')
```
Comment (helloCaller) [inline src/cohort-app.js:8]: After a quick google of what the passport module does, does using it mean we could potentially support people signing in with their google or facebook accounts?

File: src/knex/seeds/add_users.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
+require('dotenv').config({ path: __dirname + '/../../../.env' })
+
+exports.seed = function(knex, Promise) {
+  // Deletes ALL existing entries
+  return knex('users').del()
+  .then(function () {
+    return knex('users').insert([{
+      username: 'dev_user', // admin
+      password: process.env.DEV_USER_PASSWORD_HASH,
```
Comment (helloCaller) [inline src/knex/seeds/add_users.js:9]: Where does the password "process.env.DEV_USER_PASSWORD_HASH" get defined?

**Comment by jakemoves** [comment]
Yeah, there’s a clear pathway to that. And it might align with my desire to store as little user data as possible.

**Comment by jakemoves** [comment]
Oh derp, I’ll need to give you an updated copy for local env to work right. It’s the .env file you have In your root

**Pull request closed without merging**